### ISSUE TITLE
Packages.config not updated during upgrade

### ISSUE THREAD

**Issue by kreediam** [Sitefinity/Sitefinity-CLI#84]
Hello, thanks for this tool. I'm excited to use it to make upgrades easier.

I notice when I run `sf upgrade "D:\TestProject\SitefinityWebApp.sln" "13.0.7300"` that my packages.config file is not updated to match the version I just upgraded to. It still has references to the old version. For example:

```
<package id="Telerik.Sitefinity.All" version="12.2.7228" targetFramework="net472" />
```

I'm assuming this is incorrect since this would cause builds to fail on the build server (and for other team members), since they will not have the correct DLLs referenced by the project.

Am I missing something?

**Comment by RuzmanovDev** [comment]
The packages config should be updated. Can you elaborate a bit more on the steps to reproduce the issue?

**Comment by kreediam** [comment]
@RuzmanovDev I looked more into this last month, and I noticed that I have a nuget package source that was returning a 401 error when it is searched for the packages being installed. The process does continue on to other sources and is able to load the packages from the correct Sitefinity source, but it appears this issue is causing my packages.config to not be updated.

When I deselect the problematic package source in nuget settings and run the process again, my packages.config file is updated as expected. This isn’t directly a Sitefinity problem, but if I didn’t watch closely I wouldn’t find this issue, because the CLI does not report that an error happened. It would be great if the CLI could be updated to detect a failure from the update-package command.

Output from package manager console showing errors in opened Visual Studio instance:
![image](https://user-images.githubusercontent.com/435942/100892568-78b11580-3488-11eb-8d76-67575135fc09.png)

Output from CLI showing success:
![image](https://user-images.githubusercontent.com/435942/100892637-8797c800-3488-11eb-8642-efa2fa2c36de.png)
